Packages/ClientRuntime/Sources/Waiters/JMESValue.swift

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A value type that is used to facilitate comparison of values returned by JMESPath types, in accordance with the JMESPath spec.
```

Comment:
Can we add a link to the aforementioned JMESPath spec?

Reply:
This source file is now deleted.